.github/workflows/lint-test.yml

```diff
@@ -0,0 +1,42 @@
+# Copyright 2020 Vectorized, Inc.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.md
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0
+
+name: Lint and Test Charts
+
+on: [push, pull_request]
```

Comment:
Let's limit `push` runs to `main` branch only.

Reply:
Ok. Done